src/components/TileGroup/TileGroup.js

```diff
@@ -106,11 +111,22 @@ export default class TileGroup extends React.Component {
     }
   };
 
+  renderLegend = legend => {
```

Comment:
Would we need to render `null` to prevent the: "Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null." error? Or is this gone in newer versions of React?

Reply:
The newer version of React is happy with this.  It doesn't show any errors if the legend isn't there.